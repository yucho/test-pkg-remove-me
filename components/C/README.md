# @test-pkg-remove-me/c

Welcome to my `C` component.

## Installation

```sh
npm i @test-pkg-remove-me/c
# or with yarn
yarn add @test-pkg-remove-me/c
```

Then to use the component in your code just import it!

```js
import C from '@test-pkg-remove-me/c';
// and with css-modules
import '@test-pkg-remove-me/c/dist/main.css';
```
