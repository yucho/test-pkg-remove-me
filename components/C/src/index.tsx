import React from 'react';
import styles from './C.css';

interface CProps {
  /** The value for the component to display */
  value?: string;
}

/** A simple component. */
export const C = (props: CProps) => {
  return (
    <div className={styles.c}>
      This is a C component {props.value}
      More and more lines More and more lines More and more lines More and more
      lines More and more lines More and more lines More and more lines More and
      more lines More and more lines More and more lines More and more lines
      More and more lines More and more lines More and more lines More and more
      lines More and more lines More and more lines More and more lines More and
      more lines More and more lines
    </div>
  );
};

export default C;
