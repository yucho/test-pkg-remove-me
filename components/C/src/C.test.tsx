import React from 'react';
import { render } from '@testing-library/react';
import C from '.';

describe('C', () => {
  test('It matches the snapshot', () => {
    const { container } = render(<C />);
    expect(container).toMatchSnapshot();
  });
});
