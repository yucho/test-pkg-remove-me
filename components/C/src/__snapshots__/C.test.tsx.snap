// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`C It matches the snapshot 1`] = `
<div>
  <div
    class="c"
  >
    This is a C component 
  </div>
</div>
`;
